infrastructure/main.tf

```diff
@@ -24,9 +24,9 @@ module "snl-notes" {
   location             = "${var.location}"
   env                  = "${var.env}"
   ilbIp                = "${var.ilbIp}"
-  is_frontend          = false
+  is_frontend          = "${var.external_host_name != "" ? "1" : "0"}"
```

Comment:
Why not just ${var.external_host_name != ""}" ? Won't work?

Reply:
@radoslawlandowskikainos  because it is a count at the end. 
https://github.com/hmcts/cnp-module-webapp/blob/2bd9e164e2ba9b701b41b0f0f11f3d2a684cbe70/waf.tf line 16
![image](https://user-images.githubusercontent.com/4456120/46597605-21e66600-cae1-11e8-843b-916a7ad58312.png)